dev/manual_tests/lib/focus.dart

```diff
@@ -24,20 +24,14 @@ class DemoButton extends StatefulWidget {
 }
 
 class _DemoButtonState extends State<DemoButton> {
-  FocusNode focusNode;
-
-  @override
-  void initState() {
-    super.initState();
-    focusNode = FocusNode(
+  late final FocusNode focusNode = FocusNode(
       debugLabel: widget.name,
       canRequestFocus: widget.canRequestFocus,
-    );
-  }
+  );
 
   @override
   void dispose() {
-    focusNode?.dispose();
+    focusNode.dispose();
```

Comment:
Hm, now that I'm thinking about this, it's problematic if `dispose` lazily creates the property just to dispose it... It's likely not a problem in this case and it's a test, but let's keep that in mind going forward.